features/admin/ccs_category_admin_journey.feature

```diff
@@ -0,0 +1,80 @@
+@not-production @functional-test
+Feature: CCS admin user journey through Digital Marketplace
+
+Scenario: Setup for tests
+  Given I have test suppliers
+  And The test suppliers have declarations
+
+Scenario: As a CCS Category user I want to view G-Cloud 7 statistics
+  Given I have logged in to Digital Marketplace as a 'CCS Category' user
+  When I click 'G-Cloud 7 statistics'
+  Then I am presented with the 'G-Cloud 7' statistics page
+
+Scenario: As a CCS Category user I want to view Digital Outcomes and Specialists statistics
+  Given I have logged in to Digital Marketplace as a 'CCS Category' user
+  When I click 'Digital Outcomes and Specialists statistics'
+  Then I am presented with the 'Digital Outcomes and Specialists' statistics page
+
+Scenario: As a CCS Category user who has logged in to Digital Marketplace, I wish to search for a service by service ID
+  Given I have logged in to Digital Marketplace as a 'CCS Category' user
+  When I enter '1123456789012346' in the 'service_id' field
+  And I click the search button for 'service_id'
+  Then I am presented with the summary page for that service
+
+Scenario: As a CCS Category user who has logged in to Digital Marketplace, I wish to search for services by supplier ID
+  Given I have logged in to Digital Marketplace as a 'CCS Category' user
+  When I enter '11111' in the 'supplier_id_for_services' field
+  And I click the search button for 'supplier_id_for_services'
+  Then I am presented with the 'Services' page for the supplier 'DM Functional Test Supplier'
+  And I can see all listings ordered by lot name followed by listing name
+
+Scenario: As a CCS Category user who has logged in to Digital Marketplace, I wish to search for services by supplier ID and view a specific service
+  Given I am logged in as a 'CCS Category' and navigated to the 'Services' page by searching on supplier ID '11111'
+  When I select 'the' second listing on the page
```

Comment:
You are correct. I have made the change accordingly and it now reads as:
When I click the service name link for the second listing on the page

I removed the need to pass a value in as it's no longer required.